frontend/src/metabase/common/hooks/use-entity-list-query/use-entity-list-query.ts

```diff
@@ -52,17 +58,25 @@ export const useEntityListQuery = <TItem, TQuery = never>(
   const data = useSelector(state => getList(state, options));
   const error = useSelector(state => getError(state, options));
   const isLoading = useSelector(state => getLoading(state, options));
+  const isLoadingOrDefault = isLoading ?? enabled;
   const isLoaded = useSelector(state => getLoaded(state, options));
   const isLoadedPreviously = usePrevious(isLoaded);
   const isInvalidated = !isLoaded && isLoadedPreviously;
-
   const dispatch = useDispatch();
+
   useDeepCompareEffect(() => {
```

Comment:
question: do we use `useDeepCompareEffect` and not just `useEffect` because of `entityQuery`? other values seems can be compared by reference

Reply:
Use, because of the query